### PR TITLE
[Mod] build a overlay fullscreen navbar

### LINKED ISSUE #21: Navigation hamburger
### What?
A global component that will appear on every page. Goes in /src/App.vue
Shows a hamburger icon and when clicked a navigation menu folds/slides out.

**Does not** require creating the navigation system. This is just for component that displays navigation.
Use dummy values for foldout menu until there is a navigation system.

### Visual reference:
homepage-rework-proto-nav-hamburger

### PR DESCRIPTION
**Pull request by kimchirice**
## Rationale

Modify the design, as team agreed to go with a fullscreen overlay navbar, instead of a sidebar

## Advice for Reviewers & Testing Notes

- user click the menu icon, then should be able to see fullscreen overlayed navbar
- user click the close icon, then should be redicted to home 

## Screenshots:

- 
Screen Shot 2020-07-23 at 19 27 09
-
Screen Shot 2020-07-23 at 19 26 49

## Task Name

- Fixes #21 

## Linting Checklist
- [x] No commented code
- [x] Code Formatted nicely (Prettier)
- [x] PR your own code before you assign reviewers

### CODE REVIEW

File: client/src/App.vue [hunk 1]
```diff
@@ -1,20 +1,125 @@
...
+    };
+  }
+};
 </script>
 
-<style scoped>
-</style>
\ No newline at end of file
+<style>
+@import url('https://use.fontawesome.com/releases/v5.9.0/css/all.css');
```
Comment (promie) [inline client/src/App.vue:37]: Should we use the vue-fontawesome library that @chenhe-henry has installed globally within the app in his previous issue?

Comment (QjStout) [reply client/src/App.vue:37]: Yes, we should keep it consistent.

Comment (kimchirice) [reply client/src/App.vue:37]: I will modify those 2 icons

**Comment by kimchirice** [comment]
## summary of changes
 
- update icons with vue-fontawesome
- refactor App.vue, move navbar components into ./src/components/Navbar.vue

## screenshots
- Screen Shot 2020-07-24 at 19 42 27
- Screen Shot 2020-07-24 at 19 42 41

**Pull request merged**